doc/src/guides/solving.md

```diff
@@ -0,0 +1,28 @@
+# Solve Using SymPy
```

Comment:
Would "Solve Equations" be better here? 

Still not sure about including "using SymPy" in the title. It does feel redundant once you are on the docs page. From Google, I'm not sure if it's required since "SymPy" is already in the page title of every page in the docs.

Reply:
I changed it to "Solve Equations"--good point that "SymPy" is in the page title.